src/dev/i18n/extractors/html.js

```diff
@@ -40,6 +40,7 @@ import { createFailError } from '../../run';
  */
 const ANGULAR_EXPRESSION_REGEX = /\{\{+([\s\S]*?)\}\}+/g;
 
+const LINEBREAK_REGEX = /\n/g;
```

Comment:
Small question. Is it cross platform regexp? I mean will it work on Linux, Unix/Mac OS?

Reply:
@pavel06081991 we can use (and we actually use) only LF endings in all files, so it is cross platform